@here/harp-utils/lib/GroupedPriorityList.ts

```diff
@@ -108,15 +117,21 @@ export class GroupedPriorityList<T extends PriorityListElement> {
      * Return a sorted list of [[PriorityListGroup]]s.
      */
     get sortedGroups(): Array<PriorityListGroup<T>> {
```

Comment:
This gives you a sorted list, just like m_sortedGroups, no? What is the difference?